src/test/java/org/jenkinsci/plugins/workflow/cps/PersistenceProblemsTest.java

```diff
@@ -166,8 +166,7 @@ public void completedFinalFlowNodeNotPersisted() throws Exception {
 
             // Hack but deletes the file from disk
             CpsFlowExecution cpsExec = (CpsFlowExecution)(run.getExecution());
-            File f = new File(cpsExec.getStorageDir(), finalId+".xml");
-            f.delete();
+            Files.delete(cpsExec.getStorageDir().toPath().resolve(finalId+".xml"));
```

Comment:
This test wasn't failing, but I switched to `Files.delete` so that the test fails immediately with an exception if the file does not exist when we try to delete it.